test/webaudio/audioworklet_test_shared.inc

```diff
@@ -28,13 +42,18 @@ EM_JS(EMSCRIPTEN_WEBAUDIO_T, createTrack, (EMSCRIPTEN_WEBAUDIO_T ctxID, const ch
   var context = emscriptenGetAudioObject(ctxID);
   if (context) {
     var audio = document.createElement('audio');
+#if __wasm64__
+    // Number() wrapper is a workaround for UTF8ToString() needing a JS number
+    // and from64() not being available in EM_JS macros. Fix in UTF8ToString?
+    url = Number(url);
+#endif
     audio.src = UTF8ToString(url);
```

Comment:
Just do `UTF8ToString(Number(url));` unconditionally to avoid the `if/endif` to keep the code simple?

Reply:
I did, then I didn't, and flip-flopped over it. I'll keep it simple.